marketplace/optimizely/src/AppPage/index.js

```diff
@@ -80,15 +80,18 @@ export default class AppPage extends React.Component {
     const enabledContentTypes = this.findEnabledContentTypes(allContentTypes);
 
     // eslint-disable-next-line react/no-did-mount-set-state
-    this.setState(prevState => ({
-      allContentTypes,
-      config: {
-        contentTypes: enabledContentTypes,
-        optimizelyProjectId: currentParameters
-          ? currentParameters.optimizelyProjectId
-          : prevState.optimizelyProjectId
-      }
-    }));
+    this.setState(
+      prevState => ({
+        allContentTypes,
```

Comment:
Do we have a loading state for these?

Reply:
I didn't find any sort of custom loading logic for optimizely, it simply differentiates on which component to render based on whether an optimizely client has been provided, but that piece of logic is still needed